airgun/views/host.py

```diff
@@ -553,6 +554,70 @@ def before_fill(self, values):
         )
 
 
+class RecommendationWidget(GenericLocatorWidget):
+    """The widget representation of recommendation item."""
+
+    EXPAND_BUTTON = ".//div[contains(@class, 'expand')]"
+    NAME = ".//div/p[contains(@class, 'item-heading')]"
+    RISK_LABEL = ".//div/span[contains(@class, 'risk-label')]"
+    TEXT = ".//div[contains(@class, 'list-group-item-container')]"
+
+    @property
+    def expand_button(self):
```

Comment:
I think this should not be needed from outside world since you have created a property out of it.

You can either directly use it or create a var and use the var in expanded def.

Non-blocking comment though!